### ISSUE TITLE
ESP8266 Support

### ISSUE THREAD

**Issue by ybarigou** [eccnil/ESPNow2Mqtt#1]
Is there a straightforward way to change from ESP32 to ESP8266 ? I tried adding ESP8266 flags in Platformio.ini with no success

**Comment by thematrixincendies** [comment]
Yeah, I briefly looked over the code and it's a mess, the documentation, too.
For example, in the server example:
```cpp
#ifdef ESP32
  #include <WiFi.h>
#else
  #include <ESP8266WiFi.h>
#endif
#include "display.hpp"
#include "EspNow2MqttGateway.hpp"
#include <WiFi.h>
```
First, the ifdef is suggesting it can be compiled for the ESP8266 and the ESP32. Then, a few lines down, for no reason at all, ``WiFi.h`` is included a second time. But that does not even matter, because ``EspNow2MqttGateway.hpp`` is including ``<esp_now.h>``, which is again ESP32 only. On the ESP8266 the lib's header is called ``espnow.h``.
